tests/integration/cli/assets_test.py

```diff
@@ -0,0 +1,80 @@
+import os
+import tempfile
+
+from dusty.systems.virtualbox import asset_is_set, _run_command_on_vm
+from dusty import constants
+from dusty.source import Repo
+from ...testcases import DustyIntegrationTestCase
+from ...fixtures import assets_fixture
+
+class TestAssetsCLI(DustyIntegrationTestCase):
+    def setUp(self):
+        super(TestAssetsCLI, self).setUp()
+        assets_fixture()
+
+        local_dir = Repo('github.com/lib/a').managed_path
```

Comment:
How do we normally handle this in integration tests? I thought we had a way of throwing these in tmpdirs so they didn't clutter the real /etc/dusty